src/bower.json

```diff
@@ -7,5 +7,10 @@
     "leaflet": "^0.7.3",
     "multiple-select": "~1.1.0",
     "ramda": "0.8.0"
+  },
+  "overrides": {
+    "leaflet": {
+      "ignore": true
```

Comment:
We can include the css via gulp, I don't think @kshepard and I realized it was needed when we did gulp.